VueJS/index.js

```diff
@@ -1,13 +1,85 @@
-import List from './components/List.js'
-import ListItem from './components/ListItem.js'
 
-new Vue({
-  el: '#app',
-  data: {
-    todos: [
-      { id: '1', message: 'Foo', },
-      { id: '2', message: 'Bar', },
-      { id: '3', message: 'Baz', }
-    ]
-  }
+// import List from './components/List.js'
+// import ListItem from './components/ListItem.js'
```

Comment:
```suggestion
```
Remove code comments

Reply:
Also remove the files in `./components` if you don't use them. Don't leave dead code.